balancer/balancer.go

```diff
@@ -335,10 +335,12 @@ func (b *BalancerImpl) GetBestNode(ctx context.Context, redirectPrefixes []strin
 	return fallbackNode(fallbackAddr, fallbackPrefix, playbackID, redirectPrefixes[0], err)
 }
 
+// `playbackID`s matching the pattern `....-....-.....-....`
+var regexpPlaybackID = regexp.MustCompile(`^(?:\w{4}-){3}\w{4}$`)
+
 func fallbackNode(fallbackAddr, fallbackPrefix, playbackID, defaultPrefix string, err error) (string, string, error) {
 	// Check for `playbackID`s matching the pattern `....-....-.....-....`
-	r := regexp.MustCompile(`^(?:\w{4}-){3}\w{4}$`)
-	if r.MatchString(playbackID) {
+	if regexpPlaybackID.MatchString(playbackID) {
```

Comment:
```suggestion
	if regexpStreamKey.MatchString(playbackID) {
```

Reply:
That's still confusing because be pass `playbackID` into `fallbackNode()` function from `RedirectHandler()`... so in part of the code it's playbackID and now it would be considered stream-key?